go.mod

```diff
@@ -3,9 +3,9 @@ module github.com/equinix/metal-cli
 go 1.19
 
 require (
+	github.com/equinix-labs/metal-go v0.7.1
 	github.com/manifoldco/promptui v0.9.0
 	github.com/olekukonko/tablewriter v0.0.5
-	github.com/packethost/packngo v0.29.0
```

Comment:
We may have an easier time adopting metal-go if we do it gradually.  That would look similar to the metal-go PR for the terraform provider, where we set up _both_ a packngo client and a metal-go client: https://github.com/equinix/terraform-provider-equinix/pull/291